# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Add several locales to localedata.SUPPORTED
# DP: - ru_RU.CP1251 was requested in #225516
# DP: - uz_UZ.UTF-8 was clearly missing, uz_UZ had no UTF-8 variant
# DP: - da_DK.ISO-8859-15, en_GB.ISO-8859-15, en_US.ISO-8859-15, no_NO.UTF-8,
# DP:   no_NO and sv_SE.ISO-8859-15 are found in fedora-branch CVS branch.
# DP: Related bugs: #225516 (ru_RU.CP1251)
# DP: Dpatch author: Denis Barbier
# DP: Upstream status: not submitted
# DP: Date: 2006-01-10

---
 localedata/SUPPORTED |    5 +++++
 1 file changed, 5 insertions(+)

Index: langpack-locales/SUPPORTED
===================================================================
--- langpack-locales.orig/SUPPORTED	2012-03-06 10:14:13.496677751 +0100
+++ langpack-locales/SUPPORTED	2012-03-06 10:14:17.432677940 +0100
@@ -123,6 +123,7 @@
 en_DK/ISO-8859-1 \
 en_GB.UTF-8/UTF-8 \
 en_GB/ISO-8859-1 \
+en_GB.ISO-8859-15/ISO-8859-15 \
 en_HK.UTF-8/UTF-8 \
 en_HK/ISO-8859-1 \
 en_IE.UTF-8/UTF-8 \
@@ -138,6 +139,7 @@
 en_SG/ISO-8859-1 \
 en_US.UTF-8/UTF-8 \
 en_US/ISO-8859-1 \
+en_US.ISO-8859-15/ISO-8859-15 \
 en_ZA.UTF-8/UTF-8 \
 en_ZA/ISO-8859-1 \
 en_ZM/UTF-8 \
@@ -270,6 +272,7 @@
 ka_GE/GEORGIAN-PS \
 kk_KZ.UTF-8/UTF-8 \
 kk_KZ/PT154 \
+kk_KZ/RK1048 \
 kl_GL.UTF-8/UTF-8 \
 kl_GL/ISO-8859-1 \
 km_KH/UTF-8 \
@@ -351,6 +354,7 @@
 ru_RU.KOI8-R/KOI8-R \
 ru_RU.UTF-8/UTF-8 \
 ru_RU/ISO-8859-5 \
+ru_RU.CP1251/CP1251 \
 ru_UA.UTF-8/UTF-8 \
 ru_UA/KOI8-U \
 rw_RW/UTF-8 \
@@ -387,6 +391,7 @@
 sv_FI@euro/ISO-8859-15 \
 sv_SE.UTF-8/UTF-8 \
 sv_SE/ISO-8859-1 \
+sv_SE.ISO-8859-15/ISO-8859-15 \
 sw_KE/UTF-8 \
 sw_TZ/UTF-8 \
 ta_IN/UTF-8 \
@@ -416,6 +421,7 @@
 unm_US/UTF-8 \
 ur_IN/UTF-8 \
 ur_PK/UTF-8 \
+uz_UZ.UTF-8/UTF-8 \
 uz_UZ/ISO-8859-1 \
 uz_UZ@cyrillic/UTF-8 \
 ve_ZA/UTF-8 \
